java/src/main/java/com/google/oak/crypto/ClientEncryptor.java

```diff
@@ -52,21 +67,20 @@ public ClientEncryptor(final byte[] serializedServerPublicKey) {
    * @param associatedData the input byte array with associated data to be authenticated
    * @return a serialized {@code EncryptedRequest} message wrapped in a {@code Result}
    */
+  @Override
```

Comment:
Perhaps update the javadoc to be clear that the public key associated with this class is used for doing the encryption.

Reply:
The public key is not used for encryption.
Added a comment